packages/mobile/locales/es-419/fiatExchangeFlow.json

```diff
@@ -22,11 +22,7 @@
   "payWithExchange": "Exchange de criptomonedas",
   "selectCashOutMethod": "Recibe los fondos en:",
   "receiveOnAddress": "Dirección de cUSD/CELO",
-  "receiveWithPonto": "GCash (con Ponto)",
-  "receiveWithKotani": "M-PESA (con Kotani Pay)",
   "receiveWithBidali": "Tarjetas de regalo y recargas móviles",
-  "bitfy": "Bitfy",
-  "flowBtc": "FlowBTC",
```

Comment:
We shouldn't touch non English strings. It can create merge conflicts in Crowdin PRs.